cron-entrypoint

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
The repository root is starting to be cluttered with too many docker-specific definitions. 
Can you move them under a `docker` directory?